### DISCUSSION TITLE
add table-of-content for quick navigation in long notebook

### DISCUSSION THREAD

**Discussion by cailiang9** [microsoft/vscode-jupyter#9274] [Questions and Answers] [answered]
how about adding table-of-content for quick navigation in long notebook?

**Comment by DonJayamanne** [comment, marked as answer]
Thanks for the question, this is already possible today, here are a few ways to get to this:
* Click the icon on the toolbar `Show table of contents`
* Click the command from the command palette `Jupyter: Show table of contents`
* Or just go into the `Outline` view in the `Explorer` panel
